php/media/class-gallery.php

```diff
@@ -220,6 +226,7 @@ static function ( $dependency ) {
 	 * Register blocked editor assets for the gallery.
 	 */
 	public function block_editor_scripts_styles() {
+		$this->gallery_in_footer = true;
```

Comment:
Question: What is the intention of changing the external JS placement?

Reply:
Found a clean workaround for this!